bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/ObservationTimes.scala

```diff
@@ -0,0 +1,4 @@
+package edu.gemini.model.p1.immutable
+
+// All time amounts are in hours, so convert as necessary to nights.
+case class ObservationTimes(progTime: TimeAmount, partTime: TimeAmount, totalTime: TimeAmount)
```

Comment:
This is a good point. Yes, you're absolutely right: `totalTime` should not be a parameter.

I got case classes stuck in my head, thinking I was going to match on them at some point, but that never happens, I don't think, so this should just be a `val` in the class.